kapitan/refs/secrets/gkms.py

```diff
@@ -69,10 +69,11 @@ def from_params(cls, data, ref_params):
         """
         try:
             target_name = ref_params.kwargs['target_name']
-            target_inv = cached.inv['nodes'].get(target_name, None)
-            if target_name is None:
+            if not target_name:
                 raise ValueError('target_name not set')
-            if target_inv is None:
+
+            target_inv = cached.inv['nodes'].get(target_name, None)
+            if not target_inv:
```

Comment:
I find comparing to None more explicit and less magical, also more readable since .get() is defaulting to None